orchestra/todos/serializers.py

```diff
@@ -174,3 +123,20 @@ class Meta:
             'additional_data')
         read_only_fields = ('id',)
         list_serializer_class = TodoBulkCreateListSerializer
+
+
+class BulkTodoSerializerWithQAField(BulkTodoSerializer):
```

Comment:
any read_only field?

Reply:
Rename it to `BulkTodoSerializerWithoutQA`